README.md

```diff
@@ -80,3 +80,96 @@ Finally, all of the OpenWhisk resources can be shutdown using the
 template:
 
     oc process -f template.yml | oc delete -f -
+
+## Common Problems
+
+### Runtime error when setting `apihost` for `wsk`
```

Comment:
Couldn't we just add quotes to the command elsewhere in the README and eliminate this section? :)

Reply:
Dunno, does it work fine with bash and other shells?

Reply:
It doesn't bother bash. And if it appeases zsh, too, that's 2 down! :)

Reply:
Fixed